extensions/android-paging3/src/main/java/com/squareup/sqldelight/android/paging3/OffsetQueryPagingSource.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2016 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.sqldelight.android.paging3
+
+import androidx.paging.PagingState
+import com.squareup.sqldelight.Query
+import com.squareup.sqldelight.Transacter
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.withContext
+
+internal class OffsetQueryPagingSource<RowType : Any>(
+  private val queryProvider: (limit: Long, offset: Long) -> Query<RowType>,
+  private val countQuery: Query<Long>,
+  private val transacter: Transacter,
+  private val dispatcher: CoroutineDispatcher,
+) : QueryPagingSource<Long, RowType>() {
+
+  override val jumpingSupported = true
```

Comment:
```suggestion
  override val jumpingSupported get() = true
```

Otherwise you create a backing field with a constant value for no real purpose